signing_service/signing_service/utils.py

```diff
@@ -112,5 +115,41 @@ def __call__(  # type: ignore
 
 class Notifier(object):
 
-    def send(self) -> None:
-        pass
+    def __init__(
+        self,
+        from_email_address: str,
+        from_email_password: str,
+        to_email_addresses: list,
+    ) -> None:
+        self.from_email_address = from_email_address
+        self.from_email_password = from_email_password
+        self.to_email_addresses = to_email_addresses
+
+    def send(
+        self,
+        message: str,
+    ) -> None:
+        message_to_send = MIMEMultipart()
+        message_to_send['From'] = "Signing Service Notifier"
+        message_to_send['To'] = ','.join(self.to_email_addresses)
+        message_to_send['Subject'] = "Signing Service notification"
+        message_to_send.attach(MIMEText(message))
+        if self.from_email_address is not None and self.from_email_password is not None:
+            self.send_email(message_to_send)
+        print(message_to_send)
```

Comment:
Is this leftover or on purpose? If on purpose why not using logging?

Reply:
Yes, on purpose. We want to see that message in console, when `Notifier()` have no email. I do realize that `print()` is not the prettiest, and need to be change. But we kind of use logging already in Signing Service with same text.

Reply:
Done. Switch to logging.